pkg/controller/cluster_controller_test.go

```diff
@@ -91,12 +91,14 @@ func TestReconcile(t *testing.T) {
 			want:    ctrl.Result{Requeue: false},
 			wantErr: "failed to reconcile resource",
 		},
-		{
-			name:    "reconcile action updates owned resource successfully",
-			action:  fakeActor{},
-			want:    ctrl.Result{Requeue: false},
-			wantErr: "",
-		},
+		// TODO what the heck is this test doing and how do I fix it?
```

Comment:
@chrisseto any ideas?

Reply:
Looks like it's just a noop test? Do you have the error handy by any chance?